src/duration.ts

```diff
@@ -182,6 +182,9 @@ export function roundToSingleUnit(duration: Duration, {relativeTo = Date.now()}:
     } else if (monthsDiff < 11) {
       months = monthsDiff
       years = 0
+    } else if (monthsDiff === 11) {
+      months = 0
+      years += 1 // the old behavior: "11 months" is rounded to "1 year"
```

Comment:
so in this case if the two dates were January and December, we'd round up to a year? wouldn't we want it to be 11 months?

Reply:
> so in this case if the two dates were January and December, we'd round up to a year? wouldn't we want it to be 11 months?

The old code didn't do so due to ` if (monthsDiff < 11) { } else { year }` above

I have tried to make it support "11 months", it breaks a lot of tests.